src/pato/auth/micro.py

```diff
@@ -42,11 +42,12 @@ def __init__(
         super().__init__(**user)
 
 
-def _check_perms(data_id: str | int, endpoint: str, token: str, options: str = ""):
+def _check_perms(data_id: str | int, endpoint: str, token: str, options: dict[str, str | int | bool] = {}):
     response = requests.get(
         "".join(
-            [Config.auth.endpoint, "permission/", endpoint, "/", str(data_id), options]
+            [Config.auth.endpoint, "permission/", endpoint, "/", str(data_id)]
         ),
+        params={**options, "usersOnly": True},
```

Comment:
Is `"usersOnly"` supposed to have a hard-coded value?

Reply:
No, I had a local change which I hadn't pushed